standard-covoiturage_openapi.yaml

```diff
@@ -0,0 +1,1069 @@
+        luggageSize:
+          type: integer
+          minimum: 1
+          maximum: 5
+          description: |-
+            If driver journey, specifies the size of allowed luggages.
+            From very small (1) to very big (5).
+
+    Car:
+      type: object
+      properties:
+        model:
+          type: string
+          description: Model of the car.
+        brand:
+          type: string
+          description: Brand of the car.
+
+    Booking:
+      type: object
+      required:
+        - id
+        - driver
+        - passenger
+        - status
+        - price
+        - passengerPickupDate
+        - passengerPickupLat
+        - passengerPickupLng
+        - passengerDropLat
+        - passengerDropLng
+      properties:
+        id:
+          $ref: '#/components/schemas/bookingId'
+        driver:
+          $ref: '#/components/schemas/User'
+        passenger:
+          $ref: '#/components/schemas/User'
+        passengerPickupDate:
+          type: number
+          description: Passenger pickup datetime as a UNIX UTC timestamp in seconds.
+          format: long
+        passengerPickupLat:
+          type: "number"
+          format: "double"
+          description: Latitude of the passenger pick-up point.
+        passengerPickupLng:
+          type: "number"
+          format: "double"
+          description: Longitude of the passenger pick-up point.
+        passengerDropLat:
+          type: "number"
+          format: "double"
+          description: Latitude of the passenger drop-off point.
+        passengerDropLng:
+          type: "number"
+          format: "double"
+          description: Longitude of the passenger drop-off point.
+        passengerPickupAddress:
+          type: string
+          description: String representing the pickup-up address.
+        passengerDropAddress:
+          type: string
+          description: String representing the drop-off address.
+        status:
+          $ref: '#/components/schemas/bookingStatus'
+        duration:
+          type: integer
+          description: Carpooling duration in seconds.
+        distance:
+          type: integer
+          description: |
+            Carpooling distance in meters.
+        webUrl:
+          type: string
+          description: URL of the booking on the webservice provider platform.
+        price:
+          $ref: '#/components/schemas/Price'
+        car:
+          $ref: '#/components/schemas/Car'
+        driverJourneyId:
+          type: string
+          minLength: 1
+          maxLength: 255
+          description: ID of the Driver's journey to which the booking is related (if any). Unique given the `User`'s `operator` property.
+        passengerJourneyId:
+          type: string
+          minLength: 1
+          maxLength: 255
+          description: ID of the Passenger's journey to which the booking is related (if any). Unique given the `User`'s `operator` property.
```

Comment:
Rename "given the \`User\`'s \`operator\` property" into "given the \`**Passenger**\`'s \`operator\` property"